koku/api/iam/view/customer.py

```diff
@@ -194,7 +193,7 @@ def destroy(self, request, *args, **kwargs):
 
             @api {delete} /api/v1/customers/:id/ Delete a customer
```

Comment:
`@api {delete} /api/v1/customers/:id/ Delete a customer` --> `@api {delete} /api/v1/customers/:uuid/ Delete a customer`

Reply:
Got it.  I was wondering if these ids should be uuid.  I see you just updated the other places where :id was being used.